server/routes/cart.js

```diff
@@ -13,27 +13,27 @@ let ax = axios.create({
 
 // sku should be attached to req (it's in overview/products)
 
-router.get((req, res) => {
-  ax.get('/')
+router.route('/')
+  .get((req, res, next) => {
+    ax.get('/')
     .then((response) => {
       res.status(response.status).send(response.data);
     })
     .catch((error) => {
       console.error('\nerror in /cart ax get:\n', error);
       res.send('error in cart ax get');
     });
-});
-
-router.post((req, res) => {
-  ax.post('/', { 'sku_id': req.body.sku_id })
+  })
+  .post((req, res, next) => {
+    ax.post('/', { 'sku_id': req.body.sku_id })
     .then((response) => {
       res.status(response.status).send(response.statusText);
     })
     .catch((error) => {
       console.error('\nerror in /cart ax post:\n', error);
       res.send('error in cart ax post');
     });
-});
+  });
```

Comment:
good catch. there only needs to be 1 route - 2 methods
although the next is unnecessary as these are endpoints